src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Helpers/Conversions/ConversionHelpers.cs

```diff
@@ -51,5 +52,66 @@ public static List<AzureRmRecoveryServicesContainerBase> GetContainerModelList(I
 
             return containerModels;
         }
+
+        #endregion
+
+        #region policy
+        public static AzureRmRecoveryServicesPolicyBase GetPolicyModel(ProtectionPolicyResource hydraResponse)
+        {
+            AzureRmRecoveryServicesPolicyBase policyModel = null;
+
+            if(hydraResponse == null || hydraResponse.Properties == null)
+            {
+                throw new ArgumentException("Empty policy Hydra response");
```

Comment:
Let's have all exception and warning messages in one place. I have added a file for the same. Please make sure you move it to that file after my PR gets merged.